tests/bearlib/languages/documentation/DocumentationExtractionTest.py

```diff
@@ -135,39 +136,40 @@ def test_extract_documentation_PYTHON3(self):
-                    DocumentationComment(
-                        ("\n"
-                         "A nice and neat way of documenting code.\n"
-                         ":param radius: The explosion radius.\n"),
-                        docstyle_PYTHON3_default.markers[0],
-                        TextRange.from_values(8, 5, 11, 8)),
-                    DocumentationComment(
-                        ("\n"
-                         "Docstring with layouted text.\n"
-                         "\n"
-                         "    layouts inside docs are preserved for these "
-                         "documentation styles.\n"
-                         "this is intended.\n"),
-                        docstyle_PYTHON3_default.markers[0],
-                        TextRange.from_values(14, 1, 19, 4)),
-                    DocumentationComment(
-                        (" Docstring directly besides triple quotes.\n"
-                         "    Continues here. "),
-                        docstyle_PYTHON3_default.markers[0],
-                        TextRange.from_values(21, 1, 22, 24)),
-                    DocumentationComment(
-                        ("super\n"
-                         " nicely\n"
-                         "short"),
-                        docstyle_PYTHON3_default.markers[0],
-                        TextRange.from_values(35, 1, 37, 9)))
+        expected = (
+            DocumentationComment(
+                ("\n"
+                 "Module description.\n"
+                 "\n"
+                 "Some more foobar-like text.\n"),
+                docstyle_PYTHON3_default.markers[0],
+                TextRange.from_values(1, 1, 5, 4)),
+            DocumentationComment(
+                ("\n"
+                 "A nice and neat way of documenting code.\n"
+                 ":param radius: The explosion radius.\n"),
+                docstyle_PYTHON3_default.markers[0],
+                TextRange.from_values(8, 5, 11, 8)),
+            DocumentationComment(
+                ("\n"
+                 "Docstring with layouted text.\n"
+                 "\n"
+                 "    layouts inside docs are preserved for these "
+                 "documentation styles.\n"
+                 "this is intended.\n"),
+                docstyle_PYTHON3_default.markers[0],
+                TextRange.from_values(14, 1, 19, 4)),
+            DocumentationComment(
+                (" Docstring directly besides triple quotes.\n"
+                 "    Continues here. "),
+                docstyle_PYTHON3_default.markers[0],
+                TextRange.from_values(21, 1, 22, 24)),
+            DocumentationComment(
+                ("super\n"
+                 " nicely\n"
+                 "short"),
+                docstyle_PYTHON3_default.markers[0],
+                TextRange.from_values(35, 1, 37, 9)))
 
         self.assertEqual(
             tuple(extract_documentation(data, "PYTHON3", "default")),
```

Comment:
imo you just wrap the the bunch onto the next line, no real pep8 conformity change